lib/typescript/render/components/RichText/index.module.scss

```diff
@@ -0,0 +1,59 @@
+@import 'assets/scss/fonts.scss';
+@import 'assets/scss/colors.scss';
+
+.messageContainer {
```

Comment:
another nitpick: If every class in the file has the same prefix, the prefix doesn't add anything. It's a case of [context duplication](https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication)

Reply:
so true!